### PR TITLE
feat: storybook responsive addon

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Added responsive addon to Storybook

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -4,6 +4,30 @@ import {addReadme} from 'storybook-readme';
 import {withA11y} from '@storybook/addon-a11y';
 import withThemeSwitcher from './addons/theme-switcher-addon';
 
+const customViewports = {
+  Mobile: {
+    name: 'Mobile',
+    styles: {
+      width: '24rem',
```
Comment (richbachman) [inline .storybook/config.js]: Manually added these from the default theme. Subtracted 1 rem in order to get the correct viewport with the addon. 25rem wouldn't actually show the correct view for the media query.

Comment (TheSisb) [reply .storybook/config.js]: Can you import these values directly from our tokens package?

Comment (richbachman) [reply .storybook/config.js]: We can, but this add on creates an iframe with the width, so our media queries don't line up. The iframe with is 25rem using our smallest theme breakpoint, so `@media screen and (min-width: 25rem)` is still enabled in the styles. See screenshot:

image

If I set to 24rem, it works as expected. I could run some js to subtract 1 from our theme breakpoints. Thoughts?

Comment (TheSisb) [reply .storybook/config.js]: I'd rather not have 2 sources of truth for theme values anywhere, so I'd lean towards some computation based on those values.  However the bigger question is: is this addon helpful if the breakpoints aren't the ones users expect? How big of a deal is that 1rem offset?

Comment (richbachman) [reply .storybook/config.js]: Not a big deal to calculate that. Just pushed up that change.

Comment (richbachman) [reply .storybook/config.js]: Screenshot showing expected mobile view:
image

**Comment by richbachman** [comment]
Closing and opening revised PR.

**Pull request closed without merging**